src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
+  pageTitle="Buttons"
+>
+  <sky-docs-demo-page-summary>
+    The button classes create buttons to trigger actions from within an interface. For information about styles and implementation, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/buttons-links">buttons and links guidelines</a>.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Button classes"
```

Comment:
I wonder if we should change this heading to **Options** for consistency with the standard section headings on the Design tab. And then we could wrap the table in a `sky-docs-design-guideline` element and set a `heading` attribute on that element to `Button classes`. What do you think?